src/main/java/org/wildfly/extension/elytron/KeyStoreService.java

```diff
@@ -114,6 +114,13 @@ public void start(StartContext startContext) throws StartException {
             synched = System.currentTimeMillis();
             try (InputStream is = resolvedPath != null ? new FileInputStream(resolvedPath) : null) {
```

Comment:
Am not sure this is really the correct fix here - as resolvedPath is a File object I think we should perform required validation in advance and adjust the InputStream accordingly.

i.e. set the InputStream to null if the resolvedPath does not exist and required==false.  If required==true and it does not exist we can report an early error.

The IOException caught below could be caused by a number of issues so really we don't want to mix in the existence checking there.